notebooks/Tutorial7-Batch_avatarization.py

```diff
@@ -0,0 +1,196 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.5
+# ---
+
+# # Tutorial 7: Batch avatarization
+
+
+# In this tutorial, we will perform the avatarization on data batches. This can be useful if you have a dataset too large to be avatarized in one shot.
+#
+#
+# This is the general workflow of the anonymization using batch.
+#
+#
+#  ![image.png](attachment:image.png)
+#
+
+# ## Connection
+
+# +
+import warnings
+
+warnings.filterwarnings("ignore")
+
+from avatars.client import ApiClient
+
+# +
+import os
+
+url = os.environ.get("AVATAR_BASE_URL")
+username = os.environ.get("AVATAR_USERNAME")
+password = os.environ.get("AVATAR_PASSWORD")
+
+# +
+# This is the client that you'll be using for all of your requests
+from avatars.models import (
+    AvatarizationJobCreate,
+    AvatarizationParameters,
+    ImputationParameters,
+)
+from avatars.lib.split import get_split_for_batch
+from typing import Any, Dict, List, Tuple
+import math
+
+import time
+
+import numpy as np
+
+from avatars.client import ApiClient
+
+from avatars.models import (
+    AvatarizationBatchJobCreate,
+    AvatarizationBatchParameters,
+    AvatarizationBatchResult,
+    AvatarizationJob,
+    PrivacyMetricsBaseParameters,
+    PrivacyMetricsBatchJobCreate,
+    PrivacyMetricsBatchParameters,
+    PrivacyMetricsJob,
+    PrivacyMetricsJobCreate,
+    PrivacyMetricsParameters,
+    PrivacyBatchDatasetMapping,
+    SignalBatchDatasetMapping,
+    SignalMetricsBaseParameters,
+    SignalMetricsBatchJobCreate,
+    SignalMetricsBatchParameters,
+)
+from avatars.models import ImputeMethod
+from avatars.api import (
+    download_sensitive_unshuffled_avatar_from_batch,
+    upload_batch_and_get_order,
+    download_avatar_dataset_from_batch_result,
+)
+
+
+from avatars.lib.split import get_split_for_batch
+
+# The following are not necessary to
+# run avatar but are used in this tutorial
+import pandas as pd
+
+# from sklearn.model_selection import train_test_split
+
+# Change this to your actual server endpoint, e.g. base_url="https://avatar.company.com"
+client = ApiClient(base_url=url)
+client.authenticate(username=username, password=password)
+
+# Verify that we can connect to the API server
+client.health.get_health()
+# -
+
+# ## Load the data
+# We will use a subset of the dataset `adult_with_missing`.
+
+df = pd.read_csv("../fixtures/adult_with_missing.csv").iloc[:1000, :]
+
+# +
+# create some batches with from the df
+
+RowLimit = 200
```

Comment:
```suggestion
row_limit = 200
```
`snake_case` for python variables